app/views/citizens/accounts/index.html.erb

```diff
@@ -1,7 +1,11 @@
 <% content_for :navigation do %>
   <%= link_to 'Back', citizens_consent_path, class: 'govuk-back-link', id: 'back' %>
 <% end %>
-<h1 class="govuk-heading-xl"><%= t '.heading_1' %></h1>
+
+<% content_for(:page_title) { t('.h1-heading') } %>
```

Comment:
Can you make the page title declaration the very first piece of code on the page. So:
```erb
<% content_for(:page_title) { t('.h1-heading') } %>
<% content_for :navigation do %>
   ...
<% end %>